### ISSUE TITLE
Order workflow

### ISSUE THREAD

**Issue by Xriuk** [allegro/allegro-api#8137]
Hello, 
I have few questions about the orders workflow.

<hr/>

From this page: https://developer.allegro.pl/tutorials/process-orders-PgPMlWDr8Cv#event-log
I read:

> FILLED_IN - this event type may occur more than once if a customer fills in shipping form several times, e.g. when the payment is not completed or was cancelled. **It is worth noticing that with this particular status data can change until the payment is completed.**

What data can be changed at this step? Could the customer add/edit/remove products in the order?
Does this data change ability applies also to the previous order status "BOUGHT"?

<hr/>

About order combination:

> If buyer makes two separate orders from the same seller and then combine them into one payment, you will retrieve a new event with new checkoutForm.id, previous checkoutForm.id will not be available anymore. To identify this situation you can rely on lineItem.id (if one lineItem.id appears in several orders, it means that buyer combined his purchases).

If I understand this correctly, if a user makes two orders, I retrieve them, then they get combined into a single one, will I find a third order with the lines merged from the previous two? And what will happen to the previous orders? Will they appear with a status "CANCELLED" or what?

<hr/>

Can we specify multiple tracking numbers for the same order line?
https://developer.allegro.pl/documentation/#operation/createOrderShipmentsUsingPOST

In case we ship different quantities with different packages

<hr/>

Thanks

**Comment by Lukasz-Zurek** [comment]
> Hello, I have few questions about the orders workflow.
> 
> From this page: https://developer.allegro.pl/tutorials/process-orders-PgPMlWDr8Cv#event-log I read:
> 
> > FILLED_IN - this event type may occur more than once if a customer fills in shipping form several times, e.g. when the payment is not completed or was cancelled. **It is worth noticing that with this particular status data can change until the payment is completed.**
> 
> What data can be changed at this step? Could the customer add/edit/remove products in the order? Does this data change ability applies also to the previous order status "BOUGHT"?

**The customer can change the shipping data. But he cannot change/remove products from his purchase. This status does not apply in 100% to "BOUGHT" status, because "FILLED_IN" can occur many times and "BOUGHT" occurs only once (per purchase).**
> 
> About order combination:
> 
> > If buyer makes two separate orders from the same seller and then combine them into one payment, you will retrieve a new event with new checkoutForm.id, previous checkoutForm.id will not be available anymore. To identify this situation you can rely on lineItem.id (if one lineItem.id appears in several orders, it means that buyer combined his purchases).
> 
> If I understand this correctly, if a user makes two orders, I retrieve them, then they get combined into a single one, will I find a third order with the lines merged from the previous two? And what will happen to the previous orders? Will they appear with a status "CANCELLED" or what?

**That previous order won't be available because it is merged into a new one, with a new identificator (checkoutForm.id).**
> 
> Can we specify multiple tracking numbers for the same order line? https://developer.allegro.pl/documentation/#operation/createOrderShipmentsUsingPOST
> 
> In case we ship different quantities with different packages

**Yes, you can.**
> 
> Thanks

**Comment by Lukasz-Zurek** [comment]
@Xriuk Yes, indeed.
